.idea/.gitignore

```diff
@@ -0,0 +1,5 @@
+# Default ignored files
```

Comment:
[Please Change]: These `.idea` files should not be committed. Be careful using `git add .` (or other add-all commands); I tend to add files/directories specifically.